messenger.rst

```diff
@@ -640,8 +640,35 @@ You can limit the worker to only process messages from specific queue(s):
     # you can pass the --queues option more than once to process multiple queues
     $ php bin/console messenger:consume my_transport --queues=fasttrack1 --queues=fasttrack2
 
-To allow using the ``queues`` option, the receiver must implement the
-:class:`Symfony\\Component\\Messenger\\Transport\\Receiver\\QueueReceiverInterface`.
+.. note::
+
+    To allow using the ``queues`` option, the receiver must implement the
+    :class:`Symfony\\Component\\Messenger\\Transport\\Receiver\\QueueReceiverInterface`.
+
+.. _messenger-message-count:
+
+Knowing the number of messages in the "queue" for transport(s)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you want to know how many messages are in the "queues",
+you can do so with the command ``messenger:stats``:
+
+.. code-block:: terminal
+
+    # displays the number of messages in the "queue" for each all transports
```

Comment:
This sentence with each and all looks confusing to me

Reply:
I agree, 'each' was not necessary